utils/create_fixture.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Quan Zhou <quan@bitergia.com>
+#
+
+
+import argparse
+from datetime import datetime, timezone
+import json
+import logging
+
+import pymysql
+
+QUERY_DESC = 'desc {};'
+QUERY_SEARCH = 'SELECT * FROM {};'
+QUERY_TABLES_SHOW = 'show tables;'
+
+STRFTIME = '%Y-%m-%d %H:%M:%S%z'
+
+logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s',
+                    level=logging.WARNING)
+logger = logging.getLogger('create_fixture')
+
+
+def main():
+    """This script creates a fixture JSON file given the old SortingHat (0.7.x)
+    database to load into the new SortingHat using Django.
+
+    Execute:
+    ```
+    $ python3 create_fixture.py localhost test_sh
+    [2021-06-10 17:29:11,461][INFO] Start creating fixture file for test_sh
+    [2021-06-10 17:29:21,252][INFO] Fixture file created: test_sh_fixture.json
+    [2021-06-10 17:29:21,252][INFO] Total duration: test_sh_fixture.json
+    ```
+
+    First you have to create the new database.
+    ```
+    mysql> CREATE DATABASE new_sh CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci;
+    ```
+
+    Load fixture JSON file using Django
+    ```
+    $ python3 manage.py loaddata test_sh_fixture.json --settings=config.settings.devel
+    Installed 148542 object(s) from 1 fixture(s)
+    ```
+    """
+    args = parse_args()
+    host = args.host
+    user = args.user
+    password = args.password
+    database = args.database
```

Comment:
You can ignore these variables an use `args.*` directly because you are passing them just once to create the connection.